.github/workflows/examples-bot.yml

```
@@ -0,0 +1,36 @@
+name: New Gno Examples Code Notifier
+
+on:
+  pull_request:
+    types: [closed]
+    branches: [main]
+    paths:
+      - 'examples/**'
+
+jobs:
+  discord-notify:
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Send message to Discord
+        env:
+          DISCORD_WEBHOOK: ${{ secrets.DISCORD_HOOK }}
+          PR_URL: ${{ github.event.pull_request.html_url }}
+          PR_TITLE: ${{ github.event.pull_request.title }}
+          PR_NUMBER: ${{ github.event.pull_request.number }}
+          PR_AUTHOR: ${{ github.event.pull_request.user.login }}
+        run: |
```

I don't want a red master if this fails. Please ensure that a failure is silenced or maximized as a warning, but not treated as an error.

---

I added a check if the curl fails. Now this should not affect master if it fails.